src/core/lib/surface/channel.cc

```diff
@@ -277,7 +275,8 @@ static grpc_call* grpc_channel_create_call_internal(
     grpc_completion_queue* cq, grpc_pollset_set* pollset_set_alternative,
     grpc_core::Slice path, absl::optional<grpc_core::Slice> authority,
     grpc_core::Timestamp deadline, bool registered_method) {
-  auto channel = grpc_core::Channel::FromC(c_channel)->Ref();
+  auto channel =
+      grpc_core::Channel::FromC(c_channel)->RefAsSubclass<grpc_core::Channel>();
```

Comment:
I don't think this actually needs `RefAsSubclass<>`, does it?

Reply:
no... the first iteration of this was renaming `Channel` and I guess this got left out of my cleanup change because of the `grpc_core::`